packages/server/src/lib/superMode.ts

```diff
@@ -5,16 +5,17 @@ import { EpicTest } from '@sdc/shared/types';
 import { CountryGroupId } from '@sdc/shared/lib';
 import { logError, logInfo } from '../utils/logging';
 import { putMetric } from '../utils/cloudwatch';
+import { buildReloader, ValueReloader } from '../utils/valueReloader';
 
-const docClient = new AWS.DynamoDB.DocumentClient({ region: 'eu-west-1' });
```

Comment:
I noticed that building the docClient in the global scope causes credentials errors when running locally. This doesn't happen when it's built in the `fetchSuperModeArticles` function below